intezer_sdk/endpoint_analysis.py

```diff
@@ -198,6 +199,19 @@ def _send_scheduled_tasks_info(self):
         except BaseException:
             logger.warning(f'Endpoint analysis: {self.analysis_id}, failed to upload scheduled tasks info')
 
+    def _send_autoruns_info(self):
+        autoruns_info_path = os.path.join(self._offline_scan_directory, 'autoruns_info.json')
+        if not os.path.isfile(autoruns_info_path):
+            return
+        logger.info(f'Endpoint analysis: {self.analysis_id}, uploading autoruns info')
+        try:
+            with open(autoruns_info_path, encoding='utf-8') as f:
+                autoruns_info = json.load(f)
+            self._scan_api.send_autoruns_info(autoruns_info)
+        except BaseException:
```

Comment:
Change the BaseException to regular Exception (in all of the places). `BaseException` catches `KeyboardInturpt` which is raise when typing ctrl+c